pandas/tests/arithmetic/test_numeric.py

```diff
@@ -135,10 +135,14 @@ def test_div_td64arr(self, left, box_cls):
         result = right // left
         tm.assert_equal(result, expected)
 
-        with pytest.raises(TypeError):
+        msg = (
+            "Cannot divide [A-z0-9]+ data by [A-z0-9]+|"
+            "Cannot divide [A-z0-9]+ by [A-z0-9]+"
```

Comment:
Instead of testing both variations, can we make them consistent? (or otherwise just test "Cannot divide")